citools/console/commands/config.py

```diff
@@ -0,0 +1,22 @@
+import os
+import subprocess
+
+from cleo import Command
+from mozci.configuration import config
+
+
+class ConfigCommand(Command):
+    """
+    open mozci configuration file in editor
+
+    config
+    """
+
+    def handle(self):
+        path = config.path
+        editor = os.environ.get("EDITOR")
+
+        if editor is not None:
+            subprocess.run([editor, path])
+        else:
+            self.line(f'{"Error: editor not found"}')
```

Comment:
It would be helpful to tell people that they need to set the `$EDITOR` variable here, maybe something like:
`Error: set the $EDITOR environment variable`

Reply:
Done